app/helpers/application_helper.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'oj'
```

Comment:
These requires should not be necessary, since it's included in the Gemfile. Unless it has the `require: false` option, each of those gems will be required when rails boots.